README.md

```diff
@@ -1,41 +1,23 @@
-    * [Fetching and storing session tokens](#fetching-and-storing-session-tokens)
-    * [Requesting authenticated resources](#requesting-authenticated-resources)
-    * [Marking shops as uninstalled on app/uninstalled webhook](#marking-shops-as-uninstalled-on-appuninstalled-webhook)
-
-## Use Turbolinks to convert a multi-page app
-
-[Turbolinks][2] is a JavaScript library that allows your app to behave as if it were a single-page app.
-
-If you have a multi-page server-side rendered (SSR) app and you want to use session token-based authentication, but you are unsure or not yet ready to convert your app to a single-page app, then you can still use session tokens by converting your app to use Turbolinks.
+A demo app created using Rails, App Bridge, and Turbolinks for the Shopify tutorial [Authenticate server-side rendered apps with session tokens using App Bridge](https://shopify.dev/tutorials/authenticate-server-side-rendered-apps-with-session-tokens-app-bridge-turbolinks).
 
 ## Quick start
 
 To run this app locally, you can clone this repository and do the following.
 
-1. Create a `.env` file to specify your `Shopify API Key` and `Shopify API Secret` available from your partners dashboard.
+1. Create a `.env` file to specify this app's `API key` and `API secret key` app credentials that can be found in the Shopify Partners dashboard.
 
 ```
-SHOPIFY_API_KEY='YOUR API KEY FROM SHOPIFY PARTNERS DASHBOARD'
-SHOPIFY_API_SECRET='YOUR API SECRET KEY FROM SHOPIFY PARTNERS DASHBOARD'
-SHOPIFY_DOMAIN='YOUR SHOPIFY DOMAIN - DEFAULT myshopify.com'
+SHOPIFY_API_KEY=<The API key app credential specified in the Shopify Partners dashboard>
+SHOPIFY_API_SECRET=<The API secret key app credential specified in the Shopify Partners dashboard>
+APP_URL=<The app URL specified in the Shopify Partners dashboard>
 ```
 
-> __Note:__ If you do not have a Shopify API Key or Shopify API Secret, see the following sections of the [Build a Shopify App with Node and React][7] guide:
-> 1. [Expose your dev environment][9]
-> 2. [Get a Shopify API Key and Shopify API secret key][10]
-> 3. [Add the Shopify API Key and Shopify API secret key][11]
+> __Note:__ If you do not have a Shopify API Key or Shopify API Secret, see the following sections of the [Build a Shopify App with Node and React](https://shopify.dev/tutorials/build-a-shopify-app-with-node-and-react/embed-your-app-in-shopify#get-a-shopify-api-key) guide:
+> 1. [Expose your dev environment](https://shopify.dev/tutorials/build-a-shopify-app-with-node-and-react/embed-your-app-in-shopify#expose-your-dev-environment)
+> 2. [Get a Shopify API Key and Shopify API secret key](https://shopify.dev/tutorials/build-a-shopify-app-with-node-and-react/embed-your-app-in-shopify#get-a-shopify-api-key)
+> 3. [Add the Shopify API Key and Shopify API secret key](https://shopify.dev/tutorials/build-a-shopify-app-with-node-and-react/embed-your-app-in-shopify#add-the-shopify-api-key)
```

Comment:
This link points to an example that names the secret environment variable `SHOPIFY_API_SECRET_KEY` and not `SHOPIFY_API_SECRET`. Can you add a note that accounts for this?

Reply:
Oh right, that's the source of confusion for this 🤦 

Adding a note!

Reply:
See latest update.